content/_index.de.md

```diff
@@ -31,21 +31,21 @@ Diese grundlegenden Werte und Prinzipien sind:
 
 * Wir können erfolgreich sein und gleichzeitig vielfältige, erfüllende Leben führen. Unser Erfolg und unser Wert ist nicht davon abhängig, ob wir all unsere Energie in die Entwicklung von Software stecken.
 
-* Wir haben die Verpflichtung unsere - wie auch immer subtil - privilegierten Positionen zu nutzen, um das Leben Anderer zu verbessern.
+* Wir haben die Verpflichtung unsere – wie subtil auch immer – privilegierten Positionen zu nutzen, um das Leben Anderer zu verbessern.
 
 * Wir müssen Raum schaffen für Menschen, die nicht so sind wie wir, damit sie in unseren Bereich einsteigen und darin erfolgreich sind. Das bedeutet nicht nur, diese Menschen in den IT-Bereich einzuladen, sondern auch sicherzustellen, dass sie darin Unterstützung und Ermächtigung finden.
 
-* Wir haben eine moralische Verantwortung die Arbeit an Software zu verweigern, die das Wohl anderer Menschen negativ beeinträchtigt.
+* Wir haben eine moralische Verantwortung die Arbeit an Software zu verweigern, die das Wohl anderer Menschen beeinträchtigt.
 
-* Wir erkennen den Wert von Mitwirkenden, die nicht-technische Beiträge leisten, als gleichwertig zu dem Wert Mitwirkender an, die technische Beiträge liefern.
+* Wir erkennen den Wert von Mitwirkenden, die nicht-technische Beiträge leisten, als gleichwertig zu dem jener Mitwirkenden an, die technische Beiträge liefern.
 
 * Wir verstehen, dass die Arbeit in unserem Feld kein Recht, sondern ein Privileg ist. Der negative Einfluss toxischer Menschen am Arbeitsplatz und in der darüber hinausgehenden Gemeinschaft wird nicht durch deren technische Beiträge wieder wettgemacht.
 
-* Wir widmen uns der Ausübung von Mitgefühl, anstelle von Verachtung. Wir weigern uns jedoch, andere Menschen aufgrund der Wahl ihrer Werkzeuge, Methoden oder Programmiersprachen herunterzumachen.
+* Wir widmen uns der Ausübung des Mitgefühls, anstatt der Verachtung. Wir weigern uns jedoch, andere Menschen aufgrund der Wahl ihrer Werkzeuge, Methoden oder Programmiersprachen herunterzumachen.
 
-* Der IT-Bereich begrüßt technischen Wandel und wird bereichert, indem auch sozialer Wandel akzeptiert wird.
+* Der IT-Bereich begrüßt technischen Wandel und wird bereichert, indem er auch sozialen Wandel akzeptiert.
 
-* Wir streben danach, unsere Werte in allem, was wir tun, widerspiegeln zu lassen. Wir sind uns darüber im Klaren, dass Werte, die angepriesen, aber nicht in die Tat umgesetzt werden, in erster Linie gar keine Werte sind.
+* Wir streben danach, unsere Werte in allem, was wir tun, widerzuspiegeln. Wir sind uns darüber im Klaren, dass angepriesene aber nicht praktizierte Werte, in erster Linie gar keine Werte sind.
```

Comment:
`dass Werte, die angepriesen, aber nicht praktiziert werden, gar keine Werte sind. ` (removes 'in erster Linie', which isn't in the original and somehow weakens the statement, and makes some adjectives verbs)

Reply:
Yes, sounds also good :+1: